tests/scripts/thread-cert/border_router/test_plat_udp_accessiblity.py

```diff
@@ -132,8 +132,7 @@ def _test_srp_server(self, client, server_addr, server_port):
         self.assertEqual(client.srp_client_get_host_state(), 'Registered')
 
         # check if the SRP client can remove from the SRP server
-        client.srp_client_remove_host('host1')
-        client.srp_client_remove_service('ins1', '_ipp._tcp')
+        client.srp_client_remove_host()
```

Comment:
What was the problem with the original code?

Reply:
Please read https://github.com/openthread/openthread/pull/6927#issuecomment-899999114.

Reply:
Good catch !